### PR TITLE
Fix the issues surfaced after changing the schema

### PR DESCRIPTION
**Pull request by yenanw**
Every day the schema gets change a little bit, and every day I die a little inside. At least the schema is somewhat fixed now and all API requests work as intended. 🙃

### CODE REVIEW

**Comment by Puh00** [comment]
Cannot POST invoice to heroku la

**Comment by yenanw** [comment]
> 
> 
> Cannot POST invoice to heroku la

i know la

File: backend/src/main/java/net/finman/dao/InvoiceDaoImpl.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public void createInvoice(Invoice inv) throws ResourceNotCreatedException {
     private String generateOcr(){
         long ocr;
         SplittableRandom rng = new SplittableRandom();
-        ocr = rng.longs(1, 1000000000, 9999999999l).sum();  
+        ocr = rng.longs(1, 1000000000, 9999999999L).sum();
```
Comment (yenanw) [inline backend/src/main/java/net/finman/dao/InvoiceDaoImpl.java:68]: apparently codefactor hates it when you use lower-case

**Review by Puh00 — APPROVED** [review]
lgtm

**Pull request merged**